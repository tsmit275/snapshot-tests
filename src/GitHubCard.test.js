

import React from 'react'
import renderer from 'react-test-renderer'
import GitHubCard from './GithubCard'


test('GitHubCard snapshot', () => {
  const tree = renderer.create(<GitHubCard />).toJSON()
  expect(tree).toMatchSnapshot()
})
