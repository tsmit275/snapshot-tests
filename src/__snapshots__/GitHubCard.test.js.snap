// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHubCard snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      tsmit275
    </div>
    <p
      className="card-text"
    >
      <img
        alt="Your Photo"
        src="your_photo_url"
      />
      <p>
        Future Cybersecurity Professional
      </p>
    </p>
  </div>
</div>
`;
